src/lib/data/test.tsx

```diff
@@ -113,6 +113,13 @@ describe('useAppConfig', () => {
 });
 
 describe('useSettings', () => {
+    afterEach(() => {
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+        window.localStorage.__proto__.getItem = () => null;
```

Comment:
Should `localStorage` be mocked here or does jest do that automatically when you use it?

Reply:
@lamflam ?

Reply:
This is the only way I could find to mock it out.